x-pack/plugins/actions/server/builtin_action_types/servicenow/validators.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { validateCommonConfig, validateCommonSecrets } from '../case/validators';
+import { ConnectorValidation } from '../case/types';
+
+export const validate: ConnectorValidation = {
```

Comment:
Later on, we can create a generic validator so we do not re-do it all the time for every incident  management services

Reply:
Created an issue for that. https://github.com/elastic/kibana/issues/64506